### PR TITLE
fix: support executemany

### PR DESCRIPTION
**Pull request by zeromake**
## run example
error: aiomysql doesn't support executemany

``` python
import asyncio
import sqlalchemy as sa
from sqlalchemy.sql.ddl import CreateTable, DropTable
from aiomysql.sa import create_engine
from sqlalchemy.sql.expression import bindparam

metadata = sa.MetaData()
User = sa.Table(
    'user',
    metadata,
    sa.Column(
        'id',
        sa.Integer,
        autoincrement=True,
        primary_key=True,
        nullable=False,
    ),
    sa.Column(
        'name',
        sa.String(16),
        nullable=False,
    )
)

async def test_executemany(loop):
    engine = await create_engine(
         user="aiomysql",
         db="test_pymysql",
         host="127.0.0.1",
         password="mypass",
         port=3306,
         loop=loop,
         echo=True,
    )
    async with engine.acquire() as conn:
        await conn.execute(CreateTable(User))
        await conn.execute(User.insert().values([{"name": "test1"}, {"name": "test2"}]))
        async with conn.execute(User.select()) as cursor:
            user = await cursor.fetchone()
            assert user.id == 1
            assert user.name == "test1"
            user = await cursor.fetchone()
            assert user.id == 2
            assert user.name == "test2"
        sql = User.update().where(User.c.id == bindparam("id")).values({"name": bindparam("name")})
        await conn.execute(sql, [{"id": 1, "name": "t1"}, {"id": 2, "name": "t2"}])
        async with conn.execute(User.select()) as cursor:
            user = await cursor.fetchone()
            assert user.id == 1
            assert user.name == "t1"
            user = await cursor.fetchone()
            assert user.id == 2
            assert user.name == "t2"


loop = asyncio.get_event_loop()
loop.run_until_complete(test_executemany(loop))

```

### CODE REVIEW

File: tests/sa/test_sa_connection.py [hunk 1]
```diff
@@ -269,10 +270,31 @@ def test_raw_insert_with_named_params(self):
...
+                tbl.update().where(
+                    tbl.c.id == bindparam("id")
+                ).values(
+                    {"name": bindparam("name")}
+                ),
+                [
+                    {"id": 2, "name": "t2"},
+                    {"id": 3, "name": "t3"}
+                ]
+            )
```
Comment (terrycain) [inline tests/sa/test_sa_connection.py]: Personally I'd do a select just to ensure the update happened

**Comment by terrycain** [comment]
@jettify your knowledge of sqlalchemy exceeds mine so you probably best suited to review this. I had a look over the diff and it looks sane though.

**Comment by zMingGit** [comment]
same requirements

File: aiomysql/sa/connection.py [hunk 2]
```diff
@@ -64,16 +64,83 @@ def execute(self, query, *multiparams, **params):
...
+            else:
+                raise exc.ArgumentError(
+                    "Don't mix sqlalchemy SELECT "
+                    "clause with positional "
+                    "parameters"
+                )
+        compiled_params = compiled.construct_params(dp)
+        processors = compiled._bind_processors
+        params = [{
+            key: (
```
Comment (webknjaz) [inline aiomysql/sa/connection.py]:
```python
            key: processors.get(key, lambda k: k)(compiled_params[key])
```

Comment (webknjaz) [reply aiomysql/sa/connection.py]: Better:

```python
        def noop(k):
            return k
        params = [{
            key: processors.get(key, noop)(compiled_params[key])
```

File: aiomysql/sa/connection.py [hunk 3]
```diff
@@ -64,16 +64,83 @@ def execute(self, query, *multiparams, **params):
...
+        )
+        self._weak_results.add(ret)
+        return ret
+
     async def _execute(self, query, *multiparams, **params):
         cursor = await self._connection.cursor()
         dp = _distill_params(multiparams, params)
         if len(dp) > 1:
-            raise exc.ArgumentError("aiomysql doesn't support executemany")
+            # raise exc.ArgumentError("aiomysql doesn't support executemany")
```
Comment (webknjaz) [inline aiomysql/sa/connection.py]: Such comment is 100% unnecessary here.

**Pull request merged**